### ISSUE TITLE
Do you have a bachelor degree ?

### ISSUE THREAD

**Issue by abudayah** [sindresorhus/ama#525]
And what do you think about developers doesn't have any degree at all ?

**Comment by CodeMan99** [comment]
See #344 (sindresorhus answered) and #463 (others answered).

Degrees are truly about your own return on investment, **both** time & money. I personally have an associates degree from Pittsburgh Technical Institute (now PTC). I personally feel if I there is need for furthering my education, I should be shooting for a Masters degree at minimum.

It also depends on the candidates you will be competing against. You may find it difficult to prove your worth to employers if you do not have a degree.

Bottom line, know your goals and find out what you need to get there. Degrees are good. Experience  is good. If you can manage to get both in a short time frame, take it.

**Comment by sindresorhus** [comment]
I don't have a bachelor degree and I personally don't see the value of it for myself. I never liked school. I prefer learning on my own. But that's just how I work. Everyone thrives differently and have different goals. Do what's best for you and your career. For example, a degree can be a requirement for certain things like immigration and teaching, and it can be beneficial if you want a job at large corporations like Google or Facebook. If you're unsure, you probably want a degree.
